homework/App.js

```diff
@@ -11,21 +11,58 @@ class App {
    * Initialization
    * @param {string} url The GitHub URL for obtaining the organization's repositories.
    */
-  async initialize(url) {
-    // Add code here to initialize your app
-    // 1. Create the fixed HTML elements of your page
-    // 2. Make an initial XMLHttpRequest using Util.fetchJSON() to populate your <select> element
 
-    const root = document.getElementById('root');
+  static renderHeaderAndSelect(container, repositories) {
+    const nav = Util.createAndAppend('nav', container, { id: 'navigation', class: 'align-center' });
+
+    Util.createAndAppend('img', nav, {
+      src: 'hyf.png',
+      alt: 'Hack Your Future Logo',
+      class: 'logo',
+    });
+
+    Util.createAndAppend('label', nav, {
+      text: 'Hack Your Future Repositories:',
+      for: 'repository-select',
+    });
+
+    const select = Util.createAndAppend('select', nav, { id: 'repository-select' });
+    repositories.sort((repo1, repo2) =>
+      repo1.name.localeCompare(repo2.name, 'en', { sensivity: 'base' }),
+    );
+
+    repositories.forEach((repository, index) => {
+      Util.createAndAppend('option', select, {
+        text: repository.name,
+        value: repository.name,
+        id: index,
+      });
+    });
+
+    const main = Util.createAndAppend('main', container, { id: 'main' });
+    Util.createAndAppend('div', main, { id: 'repository', class: 'boxes' });
+    Util.createAndAppend('div', main, { id: 'contributors' });
+
+    Repository.render(repositories[0]);
+    App.fetchContributorsAndRender(repositories[0]);
```

Comment:
Hmmm, What about 
```suggestion
    this.fetchContributorsAndRender(repositories[0]);
```
Here and in other places as well.
But can you explain why?